hw1/task3.js

```diff
@@ -0,0 +1,17 @@
+import csv from "csvtojson";
+import fs from "fs";
+import CONSTANTS from "./constants";
+
+const writeOutput = fs.createWriteStream("output2.txt");
+const source = fs

```

Comment:
you don't need source variable, just call fs

Reply:
Fixed!